docs.README.md

```diff
@@ -81,11 +81,11 @@ For a single-page reference to all application settings and configuration option
 * {@link Window#blur blur} _Window_
 * {@link Window#bringToFront bringToFront} _Window_
 * {@link System#clearCache clearCache} _System_
-* {@link Application#close close} _Application_
+* {@link Application#quit quit} _Application_
```

Comment:
This should be in alphabetical order right? Please move to before `readHtml Clipboard`